gamedata/vscript.txt

```diff
@@ -318,6 +324,18 @@
 				"linux"		"60"
 				"windows"	"60"
 			}
+			
+			"IScriptVM::CompileScript"
+			{
+				"linux"		"9"
+				"windows"	"11"
+			}
+			
+			"IScriptVM::ReleaseScript"
+			{
+				"linux"		"10"
+				"windows"	"12"
+			}
```

Comment:
Offset looks correct, but now there duplicate offsets under same name

Reply:
Should both the `CompileScript` and `ReleaseScript` offsets be used in "#default" instead? I put these for TF2 specifically because I wasn't sure if they're the same for CSGO (which I'm assuming is the game that "#default" is based on).

Reply:
Yeah put in `#default`, TF2 will use default offsets but auto-adjusts its based on ExtraOffsets